Content.Tests/DMProject/Tests/Statements/Control Flow/spawn_while_enumerating.dm

```diff
@@ -0,0 +1,10 @@
+
+//# issue 163
+
+/proc/update()
+	return
+
+/proc/RunTest()
+	for(var/i = 0; i < 2; i++)
+		spawn(0) update()
```

Comment:
This seems to be a misunderstanding of #163. The issue is about using `spawn()` while inside a `for (i in ...)` loop.